chumweb/www/static/style.css

```diff
@@ -90,6 +90,12 @@ header, header:not(.section-site-info), .section-underline{
     padding-left: 0;
 }
 
+.link-list li a {
+    padding: 0.5em 0;
+    font-weight: bold;
```

Comment:
This does not result in bold text (as I would have expected, too), at least on Firefox (ESR) 124.0.2, see screenshot in comment below.

Reply:
Was convinced it is fine by https://github.com/sailfishos-chum/sailfishos-chum.github.io/pull/28#issuecomment-2057749167.